rootfs/usr/share/www/static/styles.css

```
@@ -4,160 +4,391 @@
+    right: -90%;
+  }
+
+  100% {
+    left: 100%;
+    right: -90%;
+  }
+}
+
+@keyframes indeterminate-short {
+  0% {
+    left: -200%;
+    right: 100%;
+  }
+
+  60% {
+    left: 107%;
+    right: -8%;
+  }
+
+  100% {
+    left: 107%;
+    right: -8%;
+  }
 }
-.suggestion svg {
-  flex-shrink: 0;
+
+.block-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(150px, 1fr));
+  column-gap: 16px;
+  row-gap: 16px;
+  max-width: 560px;
+  margin: 16px auto;
 }
-.suggestion .chevron {
-  margin-left: auto;
+
+@media (max-width: 550px) {
+  .block-container {
+    grid-template-columns: 1fr;
+  }
 }
 
-.spinner {
+.block {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  font-size: 14px;
+  font-weight: 500;
+}
+
+.block .icon {
+  fill: #ffffff;
+  background: #03a9f4;
+  border-radius: 50%;
+  padding: 8px;
+  margin-bottom: 16px;
   width: 40px;
   height: 40px;
-  cursor: pointer;
-  position: relative;
-  margin: 20px auto;
+  box-sizing: border-box;
+}
+
+.block .icon.community {
+  background: #008142;
+}
+
+.block .icon.app {
+  background: #6e41ab;
 }
-.double-bounce1,
-.double-bounce2 {
+
+.block:after {
+  border-radius: 12px;
+}
+
+.link {
   width: 100%;
   height: 100%;
-  border-radius: 50%;
-  background-color: #03a9f4;
-  opacity: 0.6;
+  cursor: pointer;
+  position: relative;
+}
+
+.link:after {
+  content: "";
   position: absolute;
   top: 0;
   left: 0;
+  width: 100%;
+  height: 100%;
+  transition: background-color .3s;
+}
+
+button.icon:hover,
+.link:hover:after {
+  background-color: rgba(0, 0, 0, 0.05);
+}
 
-  animation: sk-bounce 2s infinite ease-in-out;
+button.icon:focus-visible,
+.link:focus-visible:after {
+  background-color: rgba(0, 0, 0, 0.08);
 }
-.double-bounce2 {
-  animation-delay: -1s;
+
+button.icon:active,
+.link:active:after {
+  background-color: rgba(0, 0, 0, 0.12);
 }
-@keyframes sk-bounce {
-  0%,
-  100% {
-    transform: scale(0);
-    -webkit-transform: scale(0);
-  }
-  50% {
-    transform: scale(1);
-    -webkit-transform: scale(1);
-  }
+
+a {
+  text-decoration: none;
+  color: currentColor;
+}
+
+.footer {
+  max-width: 560px;
+  box-sizing: border-box;
+  margin: auto;
+  padding: 0 16px;
+  font-size: 14px;
+  text-align: right;
 }
 
 pre {
   display: block;
   margin: auto;
-  max-width: 450px;
-  padding-bottom: 8px;
+  margin-bottom: 16px;
   text-align: left;
   white-space: pre-line;
+  border: 1px solid rgba(0, 0, 0, 0.12);
+  border-radius: 8px;
+  padding: 16px;
+  font-family: Roboto Mono, monospace;
 }
+
 pre:empty {
   display: none;
 }
 
+dialog {
+  background-color: #ffffff;
+  border: none !important;
+  outline: none;
+  border-radius: 28px;
+  width: 560px;
+  width: min(560px, 90vw);
+  padding: 0;
+}
+
+dialog::backdrop {
+  background: rgba(0, 0, 0, .32);
+}
+
+dialog button {
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+}
+
+dialog button svg {
+  height: 100%;
+  width: 100%;
```

```suggestion
  height: 24px;
  width: 24px;
```
Icon size must be `24px`